### DISCUSSION TITLE
Possible disconnect bug

### DISCUSSION THREAD

**Discussion by naanlizard** [OvenMediaLabs/OvenMediaEngine#1617] [Help]
I've got a few users that occasionally get stuck in disconnect/reconnect loops with OBS.

Settings can be assumed to be random on the OBS side, but generally ~1280x720 30fps ~2500kbps x264 is known to do it occasionally.

Server settings haven't changed in forever, we're running the most recent OME version as of this post 0.16.5

OME server logs, a fairly wide net. Specifically, ctrl f for 22:12:20.680 and that should be approximately when it starts (the stream live/4 was where this was observed)
[omelog-clean.txt](https://github.com/AirenSoft/OvenMediaEngine/files/15194622/omelog-clean.txt)

OBS logs 
[obslog.txt](https://github.com/AirenSoft/OvenMediaEngine/files/15194630/obslog.txt)

Not sure where to start troubleshooting this, but lots of different people have seen it, and it's relatively new but I'm not sure exactly when it started due to it being only intermittent.

Anyone else seeing anything similar?

**Comment by bchah** [comment]
Have not observed this and can't reproduce it even by hammering start/stops via OBS. Are you able to reproduce it with a vanilla deployment of OME? If not, do you have a means of standing up your server config on some test systems to try to reproduce it there? It's a bit arduous but if you can duplicate your environment then you can reproduce it and start removing pieces of your configuration, all the way until you are at vanilla OME. Somewhere in the middle you will find the fault in the chain and it will tell you where to zoom in.

Good luck and keep us posted!

**Reply by naanlizard** [reply]
Hmm, if nobody else is seeing it it's likely something on our end, I was hoping it was a bug that snuck into OME so I wasn't responsible haha

We'll see about debugging somehow, hopefully it's software somewhere, and not a hardware or ISP problem!
